src/clj_kondo/impl/analyzer.clj

```diff
@@ -1591,6 +1591,7 @@
            protocol-ns nil
            protocol-name nil
            protocol-node nil
+           methods-meta {}
```

Comment:
didn't want to mess with `:methods`, to avoid touching other parts of the code where this is used